db/seeds.rb

```diff
@@ -212,7 +212,7 @@
 end
 
 seller = Seller.find_by(seller_id: 'shunfa-bakery')
-contact = Contact.find_or_create_by!(name: 'Apex for Youth', email: 'distributor@apexforyouth.com')
+contact = Contact.find_or_create_by!(name: 'Apex for Youth', email: ENV['SEED_DIST_EMAIL'])
```

Comment:
can you add this to the .env.example so folks know they have to fill it out to seed?

Reply:
Added in 8b2fc85